forge-gui/res/cardsfolder/upcoming/dunbarrow_revivalist.txt

```diff
@@ -0,0 +1,15 @@
+Name:Dunbarrow Revivalist
+ManaCost:1 B
+Types:Creature Human Warlock
+PT:3/1
+K:Bargain
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigBoon | TriggerDescription$ When CARDNAME enters the battlefield, you get a one-time boon with "When one or more creatures enter the battlefield under your control, create a Wicked Role token attached to one of them." 
+SVar:TrigBoon:DB$ Effect | Boon$ True | Duration$ Permanent | Triggers$ ChangeZoneTrig | SubAbility$ DBBargained | SpellDescription$ You get a one-time boon with "When one or more creatures enter the battlefield under your control, create a Wicked Role token attached to one of them."
```

Comment:
Great, so now Boons can be one-time too...?
but in the end they're effectively still just the half-assed version of a delayed trigger 🤕 
but whatever, the effect should still be clean itself up imo

Reply:
You mean exile the effect ?

Reply:
I don't know how its on Arena but wouldn't it get messy fast?
it also matters for the game now with _Underbridge Warlock_

Reply:
still overcomplicated
the OneOff$ param can take care of it (which also makes ActivationLimit redundant)

Reply:
Only doing it this way does the effect go away 

I've tried with oneoff 

![image](https://github.com/Card-Forge/forge/assets/67333662/4b5ecca6-d7ff-4074-b6ec-76f66c87ca2e)

Reply:
imo you're doing something wrong then because it works fine on e.g. _Flaming Fist Duskguard_

Reply:
Oh I see what I did wrong, thanks